src/verify_expr.ml

```diff
@@ -400,7 +400,7 @@ module VerifyExpr(VerifyProgramArgs: VERIFY_PROGRAM_ARGS) = struct
           let FuncInfo ([], fterm0, l0, k0, tparams0, rt0, xmap0, ng_callers_only0, pre0, pre_tenv0, post0, terminates0, _, _, is_virtual0, overrides0) = get_meth_info override in
           let ("this", PtrType (StructType base)) :: xmap0 = xmap0 in
           let base_term = direct_base_addr (prev_base_name, prev_base_term) base in
-          let () = check_func_header_compat l ("Method '" ^ meth_name ^ "'") ("Method implementation check of '" ^ override ^ "'") env
+          let () = check_func_header_compat l ("Method '" ^ meth_name ^ "'") (Printf.sprintf "Checking that '%s' correctly overrides '%s'" meth_name override) env

```

Comment:
Will the message now be "Method $derived: Checking that $derived correctly overrides $base”? Then I guess "Method $derived: Checking that it correctly overrides $base" would be better?